setup.cfg

```diff
@@ -1,41 +0,0 @@
-[metadata]
-name = jupyter_ydoc
-version = attr: jupyter_ydoc.__version__
-description = Document structures for collaborative editing using Ypy
-long_description = file: README.md
-long_description_content_type = text/markdown
-license = BSD 3-Clause License
-license_files = LICENSE
-author = David Brochart
-author_email = david.brochart@gmail.com
-url = https://github.com/jupyter-server/jupyter_ydoc
-platforms = Windows, Linux, Mac OS X
-keywords = jupyter ypy
-
-[bdist_wheel]
-universal = 1
-
-[options]
-include_package_data = True
-packages = find:
-python_requires = >=3.7
-
-install_requires =
-  setuptools
```

Comment:
Looks like this PR dropped `setuptools` as a dependency? But there is still an import of `pkg_resources` on the latest `main`:

https://github.com/jupyter-server/jupyter_ydoc/blob/1c9fc7d3deacb1cf4468f8693ed6f94eec4d310d/jupyter_ydoc/__init__.py#L1

Reply:
Thanks for catching this!